.github/workflows/rust-docker.yml

```diff
@@ -27,7 +27,10 @@ jobs:
         run: echo "defined=true" >> $GITHUB_OUTPUT
 
   build-and-push-to-gcr:
-    runs-on: buildjet-8vcpu-ubuntu-2204
+    runs-on: ubuntu-latest
```

Comment:
why use the default runner instead of depot here? assuming to parallelize further?

Reply:
previously this task would do the build on the agent itself, now we're just triggering the build on depot's own docker build runners which are separate to the depot runners we have in e2e/rust.yml

Reply:
see: https://depot.dev/docs/container-builds/overview